submissions/fue5/fue5.md

```diff
@@ -0,0 +1,72 @@
+
+![image](https://cdn.discordapp.com/attachments/628560996717559820/1116779471148683264/array.jpg)
+*You can never have enough iron plates.*
+
+There was also the need for approval from the Factorio developer Wube Software. When asked about the legality of this whole endeavour, they showed great understanding and allowed the project to be released, provided it won't be used for commercial purposes.
+
+## The Fun Part
+
+Once the general idea and most of the prototyping was figured out during the first two months of production, it became quite easy to replicate various Factorio assets and systems. That's when the time came to really focus on the beauty of the whole thing and creation of the [main release trailer](https://www.youtube.com/watch?v=01qux-5Qx_Y&feature=youtu.be&ab_channel=Hurricane), which showcases large scale bases built using the FUE5 exporter and a lot of other cool stuff one can do within the FUE5 environment.
+
+It was necessary to build a lot of custom stuff for the large-scale factories showcased in the trailer. Just the ability to export Factorio in-game bases doesn't result in very clean and cinematic results, so there were a lot of tweaks done by hand to make it nice for the camera! It also turned out that spaghetti bases are much more fun visually than the long strips of single building optimized for high SPM.
+
+![image](https://cdn.discordapp.com/attachments/628560996717559820/1116780913586618398/fun.jpg)
+*Is this the SkyBlock I've been hearing so much about?*
+
+## How to use FUE5
+
+This project is not a game, doesn't have any user interface and is kinda involved to fully understand. That being said, you can find a full description of the process on our [GitHub page](https://github.com/FUE5BASE/FUE5). Apart from installing UE5 and running the downloaded project, the process really comes down to simply selecting the chunk of your base you wish to export and then import the exported data to the FUE5 project as described in the detailed [How-to Guide](https://github.com/FUE5BASE/FUE5/blob/main/BaseImportGuide.md). Once you go through the process at least once, it becomes really easy to repeat.
+
+## The Future
+
+We wanted to share this project as open source via GitHub because the Factorio community is an extremely dedicated group of very creative people, and having access to such project might allow some of them to experience the world they love in a new dimension. Plus, it might give the more creative ones the ability to create things which were not possible before.
```

Comment:
```suggestion
We wanted to share this project as open source via GitHub, because the Factorio community is an extremely dedicated group of very creative people, and having access to such a project might allow some of them to experience the world they love in a new dimension. Plus, it might give the more creative ones the ability to create things which were not possible before.
```